core/src/main/scala/org/apache/spark/TaskContextImpl.scala

```diff
@@ -42,6 +42,7 @@ import org.apache.spark.util._
  */
 private[spark] class TaskContextImpl(
     val stageId: Int,
+    val stageAttemptId: Int,
```

Comment:
nit: add `override`. Since you are touching this file, could you also add `override` to `stageId` and `partitionId`.

Reply:
Will do.

Would you tell me the difference or rationale?

Reply:
it's kind of a code style standard: add `override` if it is override.

Reply:
OK then.